setup.py

```diff
@@ -41,7 +41,8 @@
     install_requires=[
         'six>=1.8.0',
         'w3lib>=1.15.0',
-        'cityhash>=0.1.7'
+        'cityhash>=0.1.7',
+        "thriftpy"
```

Comment:
thriftpy is only needed when installed with hbase and distributed stuff

Reply:
Tests broke when I didn't include it, should it be in a different section for installs?

Reply:
and yes, thtiftpy2

Reply:
Fixed